src/capi_flash.c

```diff
@@ -251,25 +251,29 @@ static int flash_wait_ready(int cfg, int cntl_reg, int cntl_remain)
 static void help(const char *prog)
 {
 	printf("Usage: %s [options]\n"
-		"      -h, --help       This help\n"
-		"      -v, --verbose    More Verbose\n"
-		"      -V, --version    Print version\n"
-		"      -q, --quiet      No messages\n"
-		"      -p, --factory    Program Factory (primary) Side (NEW)\n"
-		"      -a, --address    Flash Address (default: 0x%8.8x)\n"
-		"      -b, --blocksize  Flash Block Size (default: %d KB)\n"
-		"      -C, --card       Capi Card number (default: %d)\n"
-		"      -f, --file       File to flash\n\n", prog,
+		"	  -h, --help       This help\n"
```

Comment:
Please don't replace all occurrences of 4 spaces with a tab. That kills the formatting, as console output typically treats a tab as 8 spaces.